ddl/delete_range.go

```diff
@@ -268,12 +268,23 @@ func insertJobIntoDeleteRangeTable(ctx sessionctx.Context, job *model.Job) error
 	case model.ActionAddIndex:
 		tableID := job.TableID
 		var indexID int64
-		if err := job.DecodeArgs(&indexID); err != nil {
+		var partitionIDs []int64
+		if err := job.DecodeArgs(&indexID, &partitionIDs); err != nil {
```

Comment:
Will it break the backward compatibility?

Reply:
@jackysp Just adding parameters does not break compatibility.